docs/releases/4.0.txt

```diff
@@ -217,7 +217,9 @@ Management Commands
 Migrations
 ~~~~~~~~~~
 
-* ...
+* The ``makemigrations`` command might generate a no-op `AlterField` operation
```

Comment:
I didn't know how exhaustive the release note should be about the change.
Feel free toreword or add to details to the message :)